apps/orders/models.py

```diff
@@ -1,45 +1,96 @@
+from apps.common.helpers import stripe, decimal_to_int_stripe
 from django.contrib.auth.base_user import AbstractBaseUser
 from django.core.validators import MinValueValidator, MaxValueValidator
 from django.db import models
+from django.db.models import Sum, F, Subquery, OuterRef, DecimalField
+from rest_framework.exceptions import ValidationError, NotFound
+from rest_framework.generics import get_object_or_404
 
 from apps.products.models import Products
 from apps.users.models import User, UserAddress
-
+from apps.common.models import BaseModel
 
 # Create your models here.
 
-class BaseModelMixin(models.Model):
-    created_at = models.DateTimeField(auto_now_add=True)
-    updated_at = models.DateTimeField(auto_now=True)
+
+class Cart(BaseModel):
+    user = models.ForeignKey(User, on_delete=models.CASCADE, related_name='carts')
+    is_archived = models.BooleanField(default=False)
 
     class Meta:
-        abstract = True
+        ordering = ['-id']
 
+    def add_item(self, item_id, count):
```

Comment:
de ce nu trimiti aici direct instanta de product ca sa nu mai faci un get in plus? + daca primest product pune numele parametrului product